src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2579,7 +2579,7 @@ out _copyLocalFiles
                             }
                         }
                     }
-                    MSBuildEventSource.Log.RarOverallStop();
+                    MSBuildEventSource.Log.RarOverallStop(_assemblyNames.Length, _assemblyFiles.Length, _resolvedFiles?.Length ?? 0, _resolvedDependencyFiles?.Length ?? 0, _copyLocalFiles?.Length ?? 0);
```

Comment:
(not crazy familiar with RAR)
`_assemblyNames` is guaranteed not to be null here? There's a null check at the beginning of the method

Reply:
I believe it is because it's initialized to an `Array.Empty`, but the null check is fast and easy so I'm just going to do it here too.